Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:8
```

Comment:
Why openjdk? In the past, I had issues with openjdk from time to time, which were resolved by using Oracle JDK.

Reply:
Ah, so it's because [we're not allowed to redistribute the Oracle JDK](https://devops.stackexchange.com/a/434).

Reply:
The "java" docker image is deprecated and the recommended replacement was "openjdk":
https://hub.docker.com/_/java/